cli/src/pools/hooks/panoptic.ts

```diff
@@ -33,7 +33,7 @@ import { useWallet } from '../../wallet.js'
 export const usePools = () => {
   const { network, client } = usePublicClient()
   const { panopticFactory, uniswapFactory } = useFactories()
-  const [pairs, setPairs] = useState<ValidatedPair[]>([])
+  const [pairs, setPairs] = useState<ValidatedPair[]>()
```

Comment:
Needs to be [] since otherwise it is nullable and you assumed in cli/src/pools/selector.tsx it is not null